cas-server-core-util/src/main/java/org/jasig/cas/util/http/SimpleHttpClientFactoryBean.java

```diff
@@ -428,6 +429,14 @@ public void setRedirectsEnabled(final boolean redirectsEnabled) {
     }
 
     /**
+     * Destroy.
+     */
+    @PreDestroy
+    public void destroy() {
+        this.executorService.shutdownNow();
```

Comment:
Good find. I'll take a look and see if shutting down the pool makes a difference. Seems like we should be doing that anyway.